osconfig_tests/test_suites/patch/test_setup.go

```diff
@@ -17,14 +17,15 @@ package patch
 import (
 	"time"
 
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/compute"
 	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/utils"
-	"google.golang.org/api/compute/v1"
+	api "google.golang.org/api/compute/v1"
```

Comment:
consider the alias `compute` in case this ever references other apis.

Reply:
Yeah, I went ahead and fixed all the import naming